modules/fluid_properties/test/tests/interfaces/nan_interface/tests

```diff
@@ -55,6 +56,7 @@
     type = 'RunApp'
     input = 'nan_interface.i'
     cli_args = 'FluidProperties/fp/emit_on_nan=warning'
+    should_crash = true
```

Comment:
why is it crashing? we allow warnings

Reply:
Even it seems we allow this as a warning, but we still get this `Nonlinear solve did not converge due to DIVERGED_FNORM_NAN iterations 0`.